### PR TITLE
Second code review!

### PR DESCRIPTION
**Pull request by Mandeln**
Thank you for yesterdays code checkup, we have corrected the mentioned issues and are kindly requesting a second code review

Matija Maksimovic & Tedros Tesfay

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,11 +2,520 @@
...
+   <li><a href="#">U.S.</a></li>
+   <li><a href="#">Politics</a></li>
+   <li><a href="#">Election Results</a></li>
+   <li><a href="#">Newyork</a></li>
+   <li><a href="#">Business</a></li>
+   <li><a href="#">Tech</a></li>
+   <li><a href="#">Science</a></li>
+   <li><a href="#">Obtituaries</a></li>
+   <li><a href="#">Today's Paper</a></li>
+   <li><a href="#">Corrections</a></li>
```
Comment (ioilmio) [inline index.html:413]: Fix this links

**Review by ioilmio — CHANGES_REQUESTED** [review]
# Hello @Mandeln @tta2yta 👋🏽 
### Good job so far but you need to implement a couple of things;
## Required changes 🔴 :

### Main content
#### - Theory of inflation section is not displayed correctly, please fix it, it should be centered;

### Footer 
#### - Fix your links, they are not working correctly;

# Important:
- Fix your linters before submit again;
- Use a single pull request, don't close them;

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑 🔧 
## Hi There ✋🏽 
Please kindly implement the changes required by the previous TSE such as

- [ ] Fixing all your lint errors 
before submitting for a code review.

Feel free to reach out to me on slack if you have any questions, **_Witah Georjane_** 🤝 

Good luck on your project and Happy Coding 💻

File: index.html (hunk 1 above)
Comment (tta2yta) [reply index.html:413]: Hello. I have fixed the points you mentioned. Could you please check it?

**Comment by tta2yta** [comment]
Hello. I have fixed the points you mentioned. Could you please check it?

**Comment by tta2yta** [comment]
Hello. We have corrected the weblint and stylelint errors. Can you please check our pull request.

File: index.html [hunk 2]
```diff
@@ -2,11 +2,522 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
```
Comment (andresporras3423) [inline index.html:5]: Remove commented code

File: index.html [hunk 3]
```diff
@@ -2,11 +2,522 @@
...
+        <h1 class="headline">
+        Space Ripples Reveal Big Bang’s
+        <br>Smoking Gun<br>
+          </h1>
+          <div class="social-media-icons">
+            <a href="" ><img src="images/facebook-brands.svg" alt="facebook brand" style="width: 20px; height: 20px;"></a>
+            <a href=""><img src="images/twitter-719092_960_720.png" alt=" twitter brand" style="width: 20px; height: 20px;"></a>
+            <a href=""><img src="images/x-10-512.png" alt=" email share" style="width: 20px; height: 20px;"></a>
+            <a href="" ><img src="images/unnamed.png" alt="share more" style="width: 20px; height: 20px;"></a>
+            <a href=""><img src="images/later-icon-18.jpg" alt=" share later" style="width: 20px; height: 20px;"></a>
```
Comment (andresporras3423) [inline index.html]: Remove inline styles

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,557 @@
...
+
+iframe {
+  width: 100%;
+  height: 380px;
+}
+
+figure {
+  padding: 0;
+  margin: 0;
+}
```
Comment (andresporras3423) [inline css/style.css]: Group this selector and the selector on line 212

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Matija Maksimovic & Tedros Andemichael** 
Good job with the improvements for this project :muscle:  :muscle:
We still need to correct some issues before you can go ahead:

- [ ] The top navbar must have a a link with text on the leftmost side

| Yours | Original |
|:---:|:---:|
|image | image |


- [ ] Your footer doesn't match the original


| Yours | Original |
|:---:|:---:|
|image | image |

- [ ]  remove horizontal scrollbar from your page



- [ ] Your page is missing the Editor's picks section
- [ ] Remove inline styles from your HTML file
- [ ] Remove commented code in your HTML file
- [ ] Group selectors with the same style definition
- [ ] Your project should align elements with a combination of all three techniques (float, flex and grid).
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: index.html (hunk 2 above)
Comment (tta2yta) [reply index.html:5]: Hello. for the footer section we did as the original one. Try to see in desktop size(min-width: 1224px). For screen size (max-width: 1224px) we changes to other type of footer.

**Comment by tta2yta** [comment]
Hello. We have updated our project according to the reviews you sent. Please make a code review. Thanks for your time.

comments.

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @tta2yta and @Georjane 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [ ] Please work on Footer as previous TSE suggested.
- [ ] The NY Times logo is a link. Please add the navigation link in navabr logo([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by Mandeln** [comment]
with all due respect, both issues you mentioned are already implemented. Please take notice that the footer is responsive to page size and on smaller screens (again, just as it is on the real page) switches to the respective form. The logo is enveloped in a # href 
Thank you for your time

**Comment by tta2yta** [comment]
Hello. Sorry, you were opening the live demo of the application from the readme file. I understand the problem. We did not update the readme file with the latest deployed link. We have updated our application now, please review it. Thank you for your time.

**Comment by Mandeln** [comment]
@umairarshadbutt
@andresporras3423 
@ioilmio
 review

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @tta2yta and @Mandeln :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there's just a little change to be made before this project can be approved.

## REQUIRED

* [ ] You're expected to use [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

### Make the requested change and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Comment by tta2yta** [comment]
Hello. I have added the aside tag to the Editor's picks section. Could you please review the code then we can proceed to the next lesson. Thank you for your time.

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge !!!

**Pull request merged**